Api/Src/Repositories/ProjectRepository.ts

```diff
@@ -0,0 +1,16 @@
+import { Repository } from './Repository';
+import * as mongoose from 'mongoose';
+
+export default class ProjectRepository extends Repository { 
+    async deleteById(id: mongoose.Types.ObjectId) {
+        return this.model.deleteOne({_id: id});
+    };
+
+    async create(obj: object) {
+        return this.model.create(obj);
+    };
+
+    async updateById(id: mongoose.Types.ObjectId, obj: object) {
+        return this.model.findByIdAndUpdate(id, obj, {new: true, useFindAndModify: false, upsert: false});
+    };
```

Comment:
Not completely. I reworked them a bit to fit what I wanted to achieve in my task. 
I didn't want to change methods in Repository because I didn't want to create possible errors or conflicts. So I overrode them.